ksql-examples/src/main/java/io/confluent/ksql/datagen/DataGen.java

```diff
@@ -102,7 +111,8 @@ private static void usage() {
         + "topic=<kafka topic name> "
         + "key=<name of key column> "
         + "[iterations=<number of rows> (defaults to 1,000,000)] "
-        + "[maxInterval=<Max time in ms between rows> (defaults to 500)]"
+        + "[maxInterval=<Max time in ms between rows> (defaults to 500)] "
+        + "[propertiesFile=<file specifying Kafka client properties]"
```

Comment:
replace `]` with `>`

Reply:
Good catch, actually it needs to be `>]`